staging/dex-k8s-authenticator/values.yaml

```diff
@@ -73,6 +73,8 @@ resources: {}
 
 caCerts:
   enabled: false
```

Comment:
`enabled: false ?`

Reply:
yeah, the default is false. It is enabled by the operator if they want to use custom certs